tests/test_validators.py

```diff
@@ -344,8 +412,9 @@ def test_validate_always():
     class Model(BaseModel):
         a: str = None
 
-        @validator('a', pre=True, always=True)
-        def check_a(cls, v):
+        # TODO: do we support always? I think we now always validate always
```

Comment:
I don't have the answer to this question, but I'll draw attention to the question ☝️ @samuelcolvin

Reply:
`always` was weird on V1, e.g. we didn't run validation if the input value was `None` unless `always=True`, but it also caused validation to be fun on the default value.

Validating the default in pydantic-core, see https://github.com/pydantic/pydantic-core/issues/358

Reply:
Got it, seems like we're not going to support it in V2 but I'll leave this test commented out until we https://github.com/pydantic/pydantic-core/issues/358 gets done